module-services/service-cellular/service-cellular/VolteState.hpp

```diff
@@ -5,12 +5,16 @@
 
 namespace cellular
 {
-    enum class VolteState
+    struct VolteState
     {
-        On,
-        Off,
-        SwitchingToOff,
-        SwitchingToOn,
-        Undefined
+        enum class Enablement
+        {
+            On,
+            Off,
+            SwitchingToOff,
+            SwitchingToOn,
+            Undefined
+        } enablement   = Enablement::Undefined;
+        bool permitted = true; // TODO: MOS-825, now available by default for testing purposes
```

Comment:
Let's not add TODO comments in the production code :)

Reply:
Yes, but how are we supposed not to forget that this is only a partial work and must be finished with this key part in case someone gets hit by a bus? :)

Reply:
OK, no time for a discussion now